pages/dkp/konvoy/2.2/dkp-upgrade/index.md

```diff
@@ -9,6 +9,18 @@ beta: false
 
 The DKP upgrade represents an important step of your environment's lifecycle, as it ensures that you are up-to-date with the latest features and can benefit from the most recent improvements, enhanced cluster management, and better performance. This section describes how to upgrade your networked, air-gapped, or on-prem environment to the latest version of DKP.
 
+
+## Upgrade paths
+
+Verified upgrade paths are as follows:
+
+| To    | From | 1.8.4 | 1.8.5 | 2.1.0 | 2.1.1 |
+|-------|------|:-----:|:-----:|:-----:|:-----:|
+| 1.8.5 |      | Yes   |       |       |       |
+| 2.1.0 |      | Yes   | Yes   |       |       |
+| 2.1.1 |      | Yes   | Yes   | ?     |       |
+| 2.2.0 |      | ?     | ?     | ?     | ?     |
```

Comment:
You cannot upgrade directly from 1.8.* -> 2.2 because Kubernetes does not support 'skip' upgrades (i.e. you can't upgrade from K8s 1.x -> 1.[x+2], you have to first upgrade to 1.[x+1]   

DKP 1.8 had K8S 1.20,  DKP 2.2 has K8S 1.22., so the K8s versions prevent us from supporting a direct upgrade form 1.8->2.2 - you must first upgrade to 2.1.

The last two question marks should not be filled in until QA says they work, but they *should* work.

  ```suggestion
| 2.2.0 |      | No    | No    | ?     | ?     |
```